sdk/cosmosdb/Microsoft.Azure.Management.CosmosDB/src/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure CosmosDB Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure CosmosDB Resources.")]
 
-[assembly: AssemblyVersion("1.2.0.0")]
-[assembly: AssemblyFileVersion("1.2.0.0")]
+[assembly: AssemblyVersion("1.3.0.0")]
```

Comment:
@erich-wang is there a way we can incorporate the preview tag in Assembly Version

Reply:
No. Aseembly version are just numbers